.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "gml"]
 	path = gml
 	url = https://github.com/ilmola/gml.git
+[submodule "rapidjson"]
```

Comment:
I've already requested a release of gml upstream, so it can be packaged properly.
Rapidjson luckily is already in the repositories (at least on Arch).

We need to keep in mind to build ssr from the system's libs, if they're available.